fog/sample-paykit/src/client.rs

```diff
@@ -218,14 +212,6 @@ impl Client {
                 err
             })?;
 
-        let tracer = tracer!();
```

Comment:
This was causing a duplicate span that the caller was already tracking, 

<img width="1280" height="1043" alt="Image" src="https://github.com/user-attachments/assets/33dd180e-a965-43c6-a808-a00b7ebf033a" />